### ISSUE TITLE
[Bug] Java tail error logs

### ISSUE THREAD

**Issue by GuguguBear** [Melledy/Grasscutter#1666]
```
root@debian:~# uname -a
Linux debian 5.10.0-16-amd64 #1 SMP Debian 5.10.127-2 (2022-07-23) x86_64 GNU/Linux
```

```
root@debian:~# java -version
openjdk version "17.0.4" 2022-07-19
OpenJDK Runtime Environment (build 17.0.4+8-Debian-1deb11u1)
OpenJDK 64-Bit Server VM (build 17.0.4+8-Debian-1deb11u1, mixed mode, sharing)
```

```
root@debian:~# mongos -version
mongos version v6.0.0
Build Info: {
    "version": "6.0.0",
    "gitVersion": "e61bf27c2f6a83fed36e5a13c008a32d563babe2",
    "openSSLVersion": "OpenSSL 1.1.1n  15 Mar 2022",
    "modules": [],
    "allocator": "tcmalloc",
    "environment": {
        "distmod": "debian10",
        "distarch": "x86_64",
        "target_arch": "x86_64"
    }
}
```

```
2022-08-16T03:13:41Z <ERROR:emu.grasscutter.Grasscutter> An IO error occurred.
java.io.IOError: java.io.IOException: Bad file descriptor
        at org.jline.keymap.BindingReader.readCharacter(BindingReader.java:170)
        at org.jline.keymap.BindingReader.readBinding(BindingReader.java:110)
        at org.jline.keymap.BindingReader.readBinding(BindingReader.java:61)
        at org.jline.reader.impl.LineReaderImpl.doReadBinding(LineReaderImpl.java:923)
        at org.jline.reader.impl.LineReaderImpl.readBinding(LineReaderImpl.java:956)
        at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:651)
        at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:468)
        at emu.grasscutter.Grasscutter.startConsole(Grasscutter.java:323)
        at emu.grasscutter.Grasscutter.main(Grasscutter.java:166)
Caused by: java.io.IOException: Bad file descriptor
        at java.base/java.io.FileInputStream.read0(Native Method)
        at java.base/java.io.FileInputStream.read(FileInputStream.java:228)
        at org.jline.utils.NonBlockingInputStreamImpl.run(NonBlockingInputStreamImpl.java:216)
        at java.base/java.lang.Thread.run(Thread.java:833)

```

The game runs normally, enters normally, and plays normally, but there is error information, I don't know if it is useful, and I am not sure if there are any steps missing from my own setup and deployment.

游戏运行正常，进入正常，游玩正常，只是有error资讯，不知道是否有用，也不确定是否我自己架设部署少了什么步骤。

**Comment by GuguguBear** [comment]
GameVersion：grasscutter-1.2.3-dev.jar

**Comment by GuguguBear** [comment]
Also I would like to know, how to set the capacity of the logs to reduce, currently it is growing by a few hundred GB a night.

**Comment by littlesuccuba** [comment]
我也遇到了这个问题，请问你解决了吗？
I also encountered this problem. Have you solved it?

**Comment by Birdulon** [comment]
Should have been fixed by 0fb7d50acd4d3851a1c9a3531240ab6e46cb7759
